src/applications/yellow-ribbon/containers/SearchResults/index.jsx

```diff
@@ -96,18 +112,57 @@ export class SearchResults extends Component {
     return endNumber - (perPage - 1);
   };
 
+  recordEventOnSearchResultClick = (school = {}) => () => {
+    // Derive the current name params.
+    const queryParams = new URLSearchParams(window.location.search);
+
+    // Derive the state values from our query params.
+    const name = queryParams.get('name') || '';
```

Comment:
How come we need to get the `name` out of the query params instead of from the `school` object?

Reply:
It's because we are needing the search query which could be different from the actual school name.